### ISSUE TITLE
How do I turn off color augmentation when traing?

### ISSUE THREAD

**Issue by bio-rat** [ultralytics/yolov5#1065]
## ❔Question
I am training a YOLO for thermal images. I don't want to augment the color when training. What should I do to turn it off?

Thanks for your help.

**Comment by Aktcob** [comment]
https://github.com/ultralytics/yolov5/blob/master/utils/datasets.py#L563

**Comment by foodeveloperbar17** [comment]
Or you can change hyperparameters file,
https://github.com/ultralytics/yolov5/blob/7220cee1d1dc1f14003dbf8d633bbb76c547000c/data/hyp.finetune.yaml#L27
hsv_h: 0
hsv_s: 0
hsv_v: 0

**Comment by bio-rat** [comment]
That's helpful. thank you!

**Comment by glenn-jocher** [comment]
> Or you can change hyperparameters file,
> https://github.com/ultralytics/yolov5/blob/7220cee1d1dc1f14003dbf8d633bbb76c547000c/data/hyp.finetune.yaml#L27
> 
> 
> hsv_h: 0
> hsv_s: 0
> hsv_v: 0

This is the best-practices solution.

**Comment by xinxin342** [comment]
@topiaruss
 How to get the output thermal diagram?
Can you teach me the specific operation?

**Comment by adhilcodes** [comment]
> ## ❔Question
> I am training a YOLO for thermal images. I don't want to augment the color when training. What should I do to turn it off?
> 
> Thanks for your help.
> 
> ## Additional context

Does it affect the accuracy(mAP in yolov4)of the moel

**Comment by adhilcodes** [comment]
Turn off **Hue**, **Saturation** and **exposure** .
You can find it in the _YOLO cfg_ file
